### PR TITLE
Add support for create/update/get Signature

### PR DESCRIPTION
**Pull request by lavaturtle**
This adds support for interacting with the Signature endpoints of the Action Network API.

[GET signature](https://actionnetwork.org/docs/v2/signatures#get)
```
client.petitions(petition_id).signatures.get(signature_id)
```

[Update signature](https://actionnetwork.org/docs/v2/signatures#put)
```
client.petitions(petition_id).signatures.update(signature_id, {comments: 'my comments'})
```

[Create signature](https://actionnetwork.org/docs/v2/record_signature_helper)
```
client.petitions(petition_id).signatures.create({identifiers: ['controlshift:123'],
                                                 comments: 'my comments',
                                                 person: {given_name: 'Alice', email_addresses: [{address: 'alice@example.com'}]}},
                                                tags: ['volunteers'])
```

### CODE REVIEW

File: lib/action_network_rest/signatures.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module ActionNetworkRest
+  class Signatures < Base
```
Comment (woodhull) [inline lib/action_network_rest/signatures.rb:2]: Should this instead be nestled under the petitions to match the URL routes, somehow?

I'm fine with this generally though, I feel like something more OO would be too heavy most likely.

**Review by woodhull — COMMENTED** [review]
I guess I'm sort of asking if the API ergonomics should be more like:

`petitions('abc123').signatures.create('alan@turing.com')`

It probably makes the ruby way more convoluted though

**Comment by lavaturtle** [comment]
hmm, interesting idea! i'll tinker with it and see if it's feasible

**Comment by woodhull** [comment]
It was totally a random idea, so if its a bunch of work, no big deal.

**Pull request merged**